htmd/molecule/molecule.py

```diff
@@ -724,8 +724,8 @@ def getDihedral(self, atom_quad):
         --------
         >>> mol.getDihedral([0, 5, 8, 12])
         """
-        from htmd.molecule.util import dihedralAngle
-        return np.deg2rad(dihedralAngle(self.coords[atom_quad, :, self.frame]))
+        from htmd.numbautil import dihedralAngle
```

Comment:
Why `numbautil` instead just `util`. Does the implementation matters?

Reply:
Eh I was collecting all numba implementations of small stuff in a single file.

Reply:
If it grows bigger or more general I'll divide it in more files. For the moment I don't care much about the name. There is already a `htmd.util` and I didn't want to add this stuff there.